glib-2.41 is stricter checking for (un)locking mutexes, and aborts()
when attempting to unrelease an unlocked mutex.

--- src/main.c.orig	2014-09-25 11:34:24.970339238 +0200
+++ src/main.c	2014-09-25 11:34:39.034443191 +0200
@@ -1442,6 +1442,7 @@ int main ( int argc, char *argv[] )
 #endif
     g_thread_init( NULL );
     gdk_threads_init ();
+    gdk_threads_enter ();
 
 #if HAVE_HAL
     /* If the user wants to mount/umount/eject a device */
